docs/getting-started.md

```diff
@@ -64,19 +71,36 @@ In that case, you can directly pass the elements in the configure method.
 
 ```python
 import aepp
-aepp.configure(org_id=my_org_id,tech_id=my_tech_id, secret=my_secret,path_to_key=my_path_to_key,client_id=my_client_id)
+aepp.configure(
+    org_id=my_org_id,
+    tech_id=my_tech_id, 
+    secret=my_secret,
+    path_to_key=my_path_to_key,
+    client_id=my_client_id,
+    environment="prod"
+)
 ```
 
 In case you do not want to use a private.key file, you can also provide the private key as a string.
 
 ```python
 import aepp
-aepp.configure(org_id=my_org_id,tech_id=my_tech_id, secret=my_secret,private_key=my_key_as_string,client_id=my_client_id)
+aepp.configure(
+    org_id=my_org_id,
+    tech_id=my_tech_id, 
+    secret=my_secret,
+    private_key=my_key_as_string,
+    client_id=my_client_id,
+    environment="prod"
+)
 ```
 
 **NOTE** : In both case, I didn't provide a `sandbox` parameter but this parameter does exist and can be used to setup a specific sandbox.\
 By default, the prod sandbox will be used.
 
+**NOTE** The `environment` parameter is optional and defaults to "prod".
```

Comment:
should we have it default to stage for safety purposes so we dont mess up the prod env by accident

Reply:
I don't think so given this library is intended to be used first by consulting & professional services which interact directly with customer orgs, and not so much internal teams which may work in different environments. Someone should use "stage" if they really know they want to do it for a specific reason.

Reply:
Yes, the main use-case are for clients and consulting, so it does not make sense to default to stage.